packages/frontend2/src/app/(new)/(other)/scaling/data-availability/_components/table/columns.tsx

```diff
@@ -0,0 +1,106 @@
+import { createColumnHelper } from '@tanstack/react-table'
+import Image from 'next/image'
+import { SentimentText } from '~/app/_components/sentiment-text'
+import { IndexCell } from '~/app/_components/table/cells/index-cell'
+import { ProjectNameCell } from '~/app/_components/table/cells/project-name-cell'
+import {
+  TypeCell,
+  TypeColumnTooltip,
+} from '~/app/_components/table/cells/type-cell'
+import { sortSentiments } from '~/app/_components/table/sorting/functions/sentiment-sorting'
+import { type ScalingDataAvailabilityEntry } from '~/server/features/scaling/types'
+
+const columnHelper = createColumnHelper<ScalingDataAvailabilityEntry>()
+
+export const columns = [
+  // TODO: Make sure this is centered
```

Comment:
TODO

Reply:
I'll do, but this is weird - I guess I'm not styling magician coz some of the index columns that are part of different tables have no styling and row denominator is perfectly centered